Add scene.delete service for dynamically created scenes (with scene.create)

tetele opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/89090
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

tetele committed:
Add scene.delete service

frenck commented on source/_integrations/scene.markdown:
```diff
@@ -167,6 +167,28 @@ automation:
+    - service: scene.delete
+      data:
+        entity_id: scene.my_scene
+
```
```suggestion
```

frenck committed:
remove unneeded newline

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

tetele commented:
Thank you very much!

tetele committed:
Update docs to match changes upstream

https://github.com/home-assistant/core/pull/89090/commits/2b691285eec92380b333970442e55af7e8869896

tetele committed:
Revert "Update docs to match changes upstream"

This reverts commit 502c4f8afc309e25e61f74fd837c4253e296dce6.
The reason is the reversal of the upstream functionality
https://github.com/home-assistant/core/pull/89090/commits/7f0108059be140081847a046a5ab9b42b6beece5

MartinHjelmare reviewed (approved):
Thanks!
